cmd/routedns/main.go

```diff
@@ -68,6 +72,10 @@ func start(opt options, args []string) error {
 	if opt.logLevel > 6 {
 		return fmt.Errorf("invalid log level: %d", opt.logLevel)
 	}
+	if opt.version {
+		printVersion()
+		os.Exit(0)
+	}
```

Comment:
Since MinArgs is set to 0 now, I think it'd be best to add a check for the number of args here. Something like:

```golang
if len(args) < 1 {
  return errors.New("not enough arguments")
}
```